src/main/resources/templates/polls/detail.twig

```diff
@@ -4,28 +4,32 @@
 {% block description %}{{ poll.title }} {{ poll.options }}{% endblock %}
 
 {% block content %}
-	<h1>{{ poll.name }}</h1>
+	<h1>{{ poll.title }}</h1>
 
-	<form method="post" action="/polls/vote/{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}" class="poll">
-		{%- for option, nVotes in poll.options %}
+	<div class="poll">
+	<form method="post" action="/polls/vote/{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}">
 		<div class="poll--vote">
-			<label for="{{ option }}">{{ option }}: {{ nVotes }}</label>
-			<input type="radio" name="option" id="{{ option }}" value="{{ option }}">
+			{%- for option, nVotes in poll.options %}
+			<label for="{{ option }}"><input type="radio" name="option" id="{{ option }}" value="{{ option }}">{{ option }} <span>{{ nVotes }} votes</span></label>
+			<div>
+				<input type="hidden" name="{{ _csrf.parameterName }}" value="{{ _csrf.token }}"/>
+			</div>
+			{%- endfor %}
+			<button type="submit">Vote</button>
 		</div>
-		<div>
-		<input type="hidden" name="{{ _csrf.parameterName }}" value="{{ _csrf.token }}"/>
-	</div>
-		{%- endfor %}
-		<button type="submit">Vote</button>
 	</form>
 	<form method="post" action="/polls/delete/{{ poll.id }}">
-		<input type="submit" value="delete">
-		<input type="hidden" name="{{ _csrf.parameterName }}" value="{{ _csrf.token }}"/>
+		<div class="poll--delete">
+			<input type="submit" value="delete">
+			<input type="hidden" name="{{ _csrf.parameterName }}" value="{{ _csrf.token }}"/>
+		</div>
 	</form>
-	<div class="social" id="social">
+	<div class="poll--social">
 		<a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fpollistics.com%2F{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}&quote={{poll.title|url_encode}}" target="_blank" rel="noopener" class="cta">share on facebook</a>
 		<a href="https://twitter.com/intent/tweet?text={{ poll.title ~ "\n\n" ~ "via Pollistics — https://pollistics.com/" | url_encode }}{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}" target="_blank" rel="noopener" class="cta">share on twitter</a>
 	</div>
+	<div class="poll--link" id="social"></div>
```

Comment:
We think so too but do you know a way to change this without losing the styling of the facebook and twitter buttons?